src/components/Layout.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Header from './Header';
+import Footer from './Footer';
+
+const Layout = ({ children }) => {
+  // Add prop validation with PropTypes
+  Layout.propTypes = {
+    children: PropTypes.node.isRequired, // Require children prop
+  };
+
+  return (
+    <>
+      <Header />
+      {children}
```

Comment:
Can we wrap children in a Box component and make it take up the full view of the page by default? We want Footer to be at the bottom of the screen.

See (https://chakra-ui.com/docs/components/box) -- probably want the height value to be 100vh

Reply:
Or a container actually: https://chakra-ui.com/docs/components/container

Look @ the example near the bottom @T0ny-X !